packages/ui/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "tsconfig/react-library.json",
-  "include": ["."],
+  "include": [".", "./tests-setup.ts"],
```

Comment:
Required so that TypeScript knows about the matchers I mentioned in `tests-setup.ts`

Reply:
is there a reason this couldn't go under the existing paths?